app/views/schools/meters/_form.html.erb

```diff
@@ -75,6 +75,12 @@
           <%= t('schools.meters.form.leave_this_blank_message') %>
         </small>
       </div>
+
+      <div class="form-group">
+        <%= form.label(:perse_api, 'Perse API') %>
```

Comment:
Could add a note similar to the `text-muted` section above to explain this will enable loading from Perse.

The `alert-warning` in this section refers to n3rgy, might want to update that to "for n3rgy and perse integrations"